inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp

```diff
@@ -44,10 +40,10 @@ MKLDNNExperimentalDetectronTopKROIsNode::MKLDNNExperimentalDetectronTopKROIsNode
         IE_THROW() << "Operation with name '" << op->get_friendly_name() <<
             "' is not an instance of ExperimentalDetectronTopKROIs from opset6.";
 
-    if (getOriginalInputsNumber() != 2 || getOriginalOutputsNumber() != 1)
+    if (inputShapes.size() != 2 || outputShapes.size() != 1)
         IE_THROW() << errorPrefix << " has incorrect number of input/output edges!";
 
-    if (op->get_input_shape(INPUT_ROIS).size() != 2 || op->get_input_shape(INPUT_PROBS).size() != 1)
+    if (getInputShapeAtPort(INPUT_ROIS).getDims().size() != 2 || getInputShapeAtPort(INPUT_PROBS).getDims().size() != 1)
```

Comment:
minor: getRank()

Reply:
PR with fix [#7885](https://github.com/openvinotoolkit/openvino/pull/7885). Thank you!